ini/ini_map.ts

```diff
@@ -130,48 +122,48 @@ export class IniMap {
           val: formatted,
         }, LineOp.Add);
       }
-      return this.comments;
     },
-    setAtKey: (
+    setTextAtKey: (
       keyOrSection: string,
       textOrKey: string,
       noneOrText?: string,
-    ): Comments => {
+    ) => {
```

Comment:
Removed the `Comments` return type because it keeps this open to possibly return back different values in the future, but let me know if I should revert this.